docs/reference/announcements-release-notes/880/880-announcements.md

```diff
@@ -42,6 +42,18 @@ Elasticsearch 8.16+ and OpenSearch 2.17+ are now supported as minimal versions t
 </div>
 <div className="release-announcement-content">
   
+#### PostgreSQL minimal supported versions
+Web Modeler and Management Identity now also supports PostgreSQL and Amazon Aurora PostgreSQL versions 16.x and 17.x.
```

Comment:
@wollefitz - did you add support for other DBs as well? I know you support more versions of Oracle/MSSQL than Identity does 🤔

Reply:
Yep, added this via https://github.com/camunda/camunda-docs/pull/6963/files#r2407706570.

Reply:
@npepinpe Btw is there any reason why Management Identity doesn't support Postgres 18 yet?

Reply:
Because it came out on the 25th of Sept., and the PR to test for compat was done before :smile: 

I assume it would also be compatible tbh, but we haven't tried it out.